Add uBiome sequencing summary spreadsheets

dhimmel opened the pull request:
Created by @eliesbik on 2017-12-13. Commit created by @dhimmel with Elies authorship information.

Spreadsheets were emailed to Daniel by Elies and unmodified besides for their file names.

@eliesbik I created a README to add some documentation for the two spreadsheets. Would you be able to briefly summarize each sheet, and I'll add these descriptions to the bullet points.

eliesbik committed:
Add uBiome sequencing summary spreadsheets

Created by @eliesbik. Commit created by @dhimmel with
Elies authorship information.

Spreadsheets were emailed to Daniel by Elies and unmodified
besides for their file names.

results.xlsx is from the attachment FratJuice_results_v2.xlsx,
emailed on 2017-12-14.

species.xlsx is from the attachment FratJuice_species_table.xlsx,
emailed on 2017-12-13.

eliesbik commented:
Description of spreadsheet “results.xlsx”: 
- Tab “Mapping” contains the metadata associated with all the samples. It links each sample (e.g., FJ5 - gut) with sample type (concentrated or not), SeqID (unique sequencing dataset number), tube ID etc. “count” lists the number of reads assigned to Bacteria, and ‘count_norm” also lists the reads assigned to Bacteria, but normalized for a total of 1 million reads. 
- Tab “AllTaxa” lists each SeqID, and the “count” and “count_norm” assigned to each taxon, on all taxonomic levels. This is listed in the long format (useful in R).
- Tab “GenusOnly” is a subset of “AllTaxa”, just listing the genus level data. 
- Tab “SpeciesOnly” is a subset of “AllTaxa”, just listing the species level data.


Description of spreadsheet “species.xlsx”: 
- Tab “SpeciesCountsTable” lists the same data as in tab “SpeciesOnly” from spreadsheet “results.xlsx”, but now in the wide format, comparable to an OTU table. The microbial species are listed in rows, and the samples are shown in columns. The absolute counts are shown. 
- Tab “SpeciesNormTable” lists the same data as in tab “SpeciesOnly” from spreadsheet “results.xlsx”, but now in the wide format, comparable to an OTU table. The microbial species are listed in rows, and the samples are shown in columns. The normalized counts are shown (normalized to 1 million reads assigned to Bacteria). 
- Tab “Top 20” shows the 20 most abundant species, as based on the normalized counts, averaged over all the samples, and also includes a bar graph.

eliesbik committed:
Doc: describe uBiome spreadsheet tabs

Descriptions from https://github.com/dhimmel/fratjuice/pull/7#issuecomment-352508067

dhimmel commented:
Merged in https://github.com/dhimmel/fratjuice/commit/8af860a459b03a8bdc81f95aa653edcacf66f565. Thanks @eliesbik for the contribution!
